wgpu-hal/src/auxil/mod.rs

```diff
@@ -65,6 +65,17 @@ pub fn align_to(value: u32, alignment: u32) -> u32 {
     }
 }
 
+pub fn align_to_u64(value: u64, alignment: u64) -> u64 {
+    if alignment.is_power_of_two() {
+        (value + alignment - 1) & !(alignment - 1)
+    } else {
+        match value % alignment {
+            0 => value,
+            other => value - other + alignment,
+        }
+    }
+}
```

Comment:
This should probably go in a `math` module in wgpu-types, we need it in `wgpu` and `wgpu-core` too

Reply:
On a similar note, it might be nice to have an `Alignment` type similar to [naga's](https://github.com/gfx-rs/naga/blob/a7b52b8663a09506dc417e9149d64abcf637f9f4/src/proc/layouter.rs#L10).